ocs_ci/utility/environment_check.py

```diff
@@ -102,6 +102,11 @@ def assign_get_values(env_status_dict, key, kind=None, exclude_labels=None):
         if app_label in exclude_labels:
             log.debug("ignoring item with app label %s: %s", app_label, item)
             continue
+        if item.get("kind") == constants.POD:
+            name = item.get("metadata").get("name")
```

Comment:
```suggestion
            name = item.get("metadata", {}).get("name", "")
```
I already saw that from some reason the query didn't content some of values, so to be 100% sure I would go with this safe query, WDYT? As in any case you will get None ti will fail with calling endswith on None object.

Reply:
Looks reasonable, just a question: Could there be a pod without a name that we don't want to miss in teardown?

Reply:
done